app/src/main/res/values-v26/ic_launcher_background.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <color name="ic_launcher_background">#FFFFFF</color>
```

Comment:
```suggestion
  <color name="ic_launcher_background">#D4D4D4</color>
```
The file you deleted (`app/src/main/res/values/ic_launcher_background.xml`) had a color value of #D4D4D4 and not #FFFFFF.